### DISCUSSION TITLE
spot intensity

### DISCUSSION THREAD

**Discussion by irelamb** [BBQuercus/deepBlink#149] [Q&A] [answered]
Still another question... How is the intensity of a spot calculated? Is it simply the sum of the intensity of each spot in the specified radius, or is it inferred in another way?

Irene

**Comment by BBQuercus** [comment, marked as answer]
It's very straightforward by just getting the sum of intensities in the given radius. Trackpy would have a slightly more fancy way but we didn't want to include another package for just one calculation. It currently uses the `get_intensities` method from `deepblink.inference`.
